modules/vault-elb/variables.tf

```diff
@@ -25,6 +25,20 @@ variable "vault_asg_name" {
 # These parameters have reasonable defaults.
 # ---------------------------------------------------------------------------------------------------------------------
 
+# Due to terraform type limitations this variable is set to be a map of any.
```

Comment:
It actually has to be `any`, rather than `map(any)`. With `map(any)`, Terraform, sadly, would still require all the values to be of the same type, which won't work here. Until that's improved, we have to fall to the lowest common denominator of `type = any` and document the real type.

Reply:
corrected